app/src/main/java/com/github/libretube/services/DownloadService.kt

```diff
@@ -60,7 +60,7 @@ class DownloadService : Service() {
         TODO("Not yet implemented")
     }
 
-    private fun downloadManager() {
+    private fun downloadManager(videoName: String) {
```

Comment:
`videoName` is reachable in that scope that means it doesn't need to get passed to the function manually.